site/_docs/history.md

```diff
@@ -54,18 +150,321 @@ other software versions as specified in gradle.properties.
+  Add SHA256, SHA512 functions (enabled in BigQuery and PostgreSQL libraries)
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5619">CALCITE-5619</a>]
+  Add TO_CHAR(<TIMESTAMP>, <STRING>) function (enabled in MySQL,Oracle,PostgreSQL libraries)
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5761">CALCITE-5761</a>]
+  Allow DECADE, CENTURY, and MILLENNIUM time units in DATE_TRUNC, TIMESTAMP_TRUNC, DATETIME_TRUNC functions
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5783">CALCITE-5783</a>]
+  Support hint for TableFunctionScan
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5593">CALCITE-5593</a>]
+  Elasticsearch adapter should support aliases
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5411">CALCITE-5411</a>]
+  `SparkSqlDialect` should support `ROLLUP` and `CUBE` aggregate functions
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5664">CALCITE-5664</a>]
+  Add CONVERT(string USING transcodingName) function, also known as TRANSLATE
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5610">CALCITE-5610</a>]
+  Add COTH, CSCH, SECH functions
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5606">CALCITE-5606</a>]
+  Add SqlLibrary.ALL
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5746">CALCITE-5746</a>]
+  Support JDK 19
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5662">CALCITE-5662</a>]
+  Allow CAST(BOOLEAN as INTEGER) (if enabled by conformance)
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-4334">CALCITE-4334</a>]
+  LITERAL_AGG, an internal aggregate function that returns a constant value
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5764">CALCITE-5764</a>]
+  Add Puffin
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5706">CALCITE-5706</a>]
+  Add class PairList
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5688">CALCITE-5688</a>]
+  Support TRUNCATE TABLE DDL statement in server parser
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5711">CALCITE-5711</a>]
+  Implement the SINGLE_VALUE aggregation in PostgreSQL Dialect
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5608">CALCITE-5608</a>]
+  Implement ASINH, ACOSH, ATANH functions
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5602">CALCITE-5602</a>]
+  Implement CSC and SEC functions
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5367">CALCITE-5367</a>]
+  Implement spatial type functions
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5160">CALCITE-5160</a>]
+  ANY/SOME, ALL operators should support collection expressions
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5403">CALCITE-5403</a>]
+  Babel parser should parse PostgreSQL's SET, RESET, BEGIN, SHOW, ROLLBACK, COMMIT commands
+
 #### Dependency version upgrade
 {: #dependency-1-35-0}
 
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5477">CALCITE-5477</a>]
+  Build with Guava 19.0
```

Comment:
This is a downgrade so maybe it is worth putting in the breaking changes section.

Reply:
Done.